arch/arm64/src/zynq-mpsoc/zynq_serial.c

```diff
@@ -1024,6 +1024,9 @@ static void zynq_uart_txint(struct uart_dev_s *dev, bool enable)
 {
   struct zynq_uart_port_s *port = (struct zynq_uart_port_s *)dev->priv;
   struct zynq_uart_config *config = &port->config;
+  irqstate_t flags;
+
+  flags = enter_critical_section();
```

Comment:
let's add spinlock to zynq_uart_port_s and use it

Reply:
will add spinlock support in next patch

Reply:
but why not in this patch